src/plugins/intel_gpu/src/kernel_selector/kernels/eltwise/eltwise_kernel_blocked_opt.cpp

```diff
@@ -314,40 +364,39 @@ JitConstants EltwiseKernel_blocked_opt::GetJitConstants(const eltwise_params& pa
 
 EltwiseKernelBase::DispatchData EltwiseKernel_blocked_opt::SetDefault(const eltwise_params& params) const {
     DispatchData dispatchData;
-    auto in_layout = params.inputs[0].GetLayout();
-    auto out_layout = params.outputs[0].GetLayout();
-    std::vector<std::vector<Tensor::DataChannelName>> dims_by_gws = {{Tensor::DataChannelName::FEATURE},
-                                                                     {Tensor::DataChannelName::X, Tensor::DataChannelName::Y},
-                                                                     {Tensor::DataChannelName::BATCH}};
-    // Global workgroup size 0: feature, 1: spatial, 2: batch
-    dispatchData.gws[0] = CeilDiv(params.outputs[0].Feature().v, GetFeatureBlockSizeFromFormat(params, 0));
-    dispatchData.gws[2] = params.outputs[0].Batch().v;
-    if (DataTensor::ChannelsCount(params.outputs[0].GetLayout()) == 5)
-        dispatchData.gws[1] = params.outputs[0].X().v * params.outputs[0].Y().v * params.outputs[0].Z().v;
-    else if (DataTensor::ChannelsCount(params.outputs[0].GetLayout()) == 4)
-        dispatchData.gws[1] = params.outputs[0].X().v * params.outputs[0].Y().v;
-    else
-        IE_ASSERT("Unexpected dimension for eltwise_blocked_opt kernel.");
 
-    // Calculate local workgroup size
-    dispatchData.lws = GetOptimalLocalWorkGroupSizes(dispatchData.gws, params.engineInfo, in_layout, out_layout, dims_by_gws);
-    if (out_layout == DataLayout::b_fs_yx_fsv4) {
-        dispatchData.lws[0] = 1;
-        dispatchData.lws[2] = 1;
-    }
+    // Instead of using multiple channel(e.g. {{FEATURE}, {SPATIAL}, {BATCH}}), this kernel uses 1 channel which contains all logical size.
+    // so that each global id can be an index of each work group.
+    // It also makes an index for fomatted GET_INDEX macro if needed(e.g. feature broadcasting, fusing).
+    KernelData kd = KernelData::Default<eltwise_params>(params);
+    dispatchData.gws = {std::max(GetGWSfromFormat(params) / GetFeatureBlockSizeFromFormat(params, 0), (size_t)1), 1, 1};
+    dispatchData.lws = GetOptimalLocalWorkGroupSizes(dispatchData.gws, params.engineInfo);
 
     return dispatchData;
 }
 
 // Local
+static inline size_t GetGWSfromFormat(const eltwise_params& params) {
```

Comment:
nit: What about `CalculateTotalWorkItemCount`? As it is 1d data, it is not yet gws..